draft-roughtime-aanchal.xml

```diff
@@ -673,6 +737,23 @@ long-term key: S3AzfZJ5CjSdkJ21ZJGbxqdYP/SoE8fXKY0+aicsehI=
           </list>
         </t>
       </section>
+      <section title="Roughtime Version Registry">
+        <t> IANA is requested to create a new registry entitled
+        &quot; Roughtime Version Registry &quot; Entries shall have the following fields:
+        <list>
+          <t> Version: a 32-bit unsigned integer </t>
+        </list>
```

Comment:
I suggest adding a reference column. In the case of version 1 it would reference `[[this memo]]`.